src/autocomplete/dataset.js

```diff
@@ -38,7 +38,9 @@ function Dataset(o) {
   this.templates = getTemplates(o.templates, this.displayFn);
 
   this.css = _.mixin({}, css, o.appendTo ? css.appendTo : {});
-  this.cssClasses = _.mixin({}, css.defaultClasses, o.cssClasses || {});
+  this.cssClasses = o.cssClasses = _.mixin({}, css.defaultClasses, o.cssClasses || {});
```

Comment:
Just saw this was also done already in other places but I cannot get why we need it here, let me know